frame/staking-rewards/src/test/mod.rs

```diff
@@ -28,6 +25,39 @@ use sp_std::collections::btree_map::BTreeMap;
 mod prelude;
 mod runtime;
 
+#[test]
+fn test_update_reward_pool() {
+	new_test_ext().execute_with(|| {
+		System::set_block_number(1);
+
+		Timestamp::set_timestamp(200000);
+
+		let pool_init_config = get_default_reward_pool();
+		assert_ok!(StakingRewards::create_reward_pool(Origin::root(), pool_init_config));
```

Comment:
Check that the `RewardPoolCreated` is emitted and get the `pool_id` from there (could extract this to a helper function in the tests, will surely be used elsewhere)

Reply:
For what I see in another test, the `pool_count()` is used as `pool_id`. Will this be enough?